src/inabox/inabox-resources.js

```diff
@@ -226,6 +249,25 @@ export class InaboxResources {
     this.passObservable_.fire();
     this.firstPassDone_.resolve();
   }
+
+  /**
+   * Builds any pending resouces if document is ready, or next element has been
+   * added to DOM.
+   * @private
+   */
+  buildReadyResources_() {
+    for (let i = this.pendingBuildResources_.length - 1; i >= 0; i--) {
+      const resource = this.pendingBuildResources_[i];
+      if (
+        this.documentReady_ ||
+        hasNextNodeInDocumentOrder(resource.element, this.ampdoc_.getRootNode())
+      ) {
+        this.pendingBuildResources_.splice(i, 1);
+        resource.build().then(() => this./*OK*/ schedulePass());
```

Comment:
the new code here schedulePass w/o waiting for doc ready. would that cause an issue?

Reply:
seems in regular AMP, we don't wait for doc ready either.